Go v1.18 breaks installation procedure in docs

Hello all,

I upgraded to Go v1.18 over the weekend, and the installation instructions under Getting Started -> Installing Ginkgo no longer work.

The following is the output I receive when attempting to run `go get github.com/onsi/ginkgo/v2/ginkgo` in the `golang:latest` container. I have achieved similar results in a Github Action:

```
# go install github.com/onsi/ginkgo/v2/ginkgo
/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.1.3/ginkgo/generators/bootstrap_command.go:9:2: missing go.sum entry for module providing package github.com/go-task/slim-sprig (imported by github.com/onsi/ginkgo/v2/ginkgo/generators); to add:
	go get github.com/onsi/ginkgo/v2/ginkgo/generators@v2.1.3
/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.1.3/ginkgo/internal/profiles_and_reports.go:12:2: missing go.sum entry for module providing package github.com/google/pprof/profile (imported by github.com/onsi/ginkgo/v2/ginkgo/internal); to add:
	go get github.com/onsi/ginkgo/v2/ginkgo/internal@v2.1.3
/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.1.3/ginkgo/labels/labels_command.go:15:2: missing go.sum entry for module providing package golang.org/x/tools/go/ast/inspector (imported by github.com/onsi/ginkgo/v2/ginkgo/labels); to add:
	go get github.com/onsi/ginkgo/v2/ginkgo/labels@v2.1.3
```

This matches the Go dev blog's [notes](https://go.dev/doc/go-get-install-deprecation) around the deprecation of `go get` as an installer. That said, replacing get with install doesn't work either.

The fix I found is to double up the get and install commands like so:
```
go get -d github.com/onsi/ginkgo/v2/ginkgo && go install github.com/onsi/ginkgo/v2/ginkgo
```

Based on my own efforts, installing GInkgo this way doesn't break Go v1.17, but I didn't test on earlier versions. I've also been unable to replicate a need to explicitly invoke `go get github.com/onsi/gomega/...`.

I'm happy to PR the change into the docs, but I didn't know if it was going to be part of a larger update in the works.

--- a/docs/index.md
+++ b/docs/index.md
@@ -35,11 +35,11 @@ In this section we cover installing Ginkgo, Gomega, and the `ginkgo` CLI.  We bo
 
 ### Installing Ginkgo
 
-Ginkgo uses [go modules](https://go.dev/blog/using-go-modules).  To add Ginkgo to your project, assuming you have a `go.mod` file setup, just `go get` it:
+Ginkgo uses [go modules](https://go.dev/blog/using-go-modules).  To add Ginkgo to your project, assuming you have a `go.mod` file setup, just `go install` it:
 
 ```bash
-go get github.com/onsi/ginkgo/v2/ginkgo
-go get github.com/onsi/gomega/...
+go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
+go install -mod=mod github.com/onsi/gomega/...
 ```
 
 This fetches Ginkgo and installs the `ginkgo` executable under `$GOBIN` - you'll want that on your `$PATH`.  It also fetches the core Gomega matcher library and its set of supporting libraries.  Note that the current supported major version of Ginkgo is `v2`.

--- a/docs/index.md
+++ b/docs/index.md
@@ -39,7 +39,7 @@ Ginkgo uses [go modules](https://go.dev/blog/using-go-modules).  To add Ginkgo t
 
 ```bash
 go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
-go install -mod=mod github.com/onsi/gomega/...
+go get github.com/onsi/gomega/...
 ```
 
 This fetches Ginkgo and installs the `ginkgo` executable under `$GOBIN` - you'll want that on your `$PATH`.  It also fetches the core Gomega matcher library and its set of supporting libraries.  Note that the current supported major version of Ginkgo is `v2`.

